proprietary/tokens/src/components/ams/avatar.tokens.json

```diff
@@ -0,0 +1,48 @@
+{
+  "ams": {
+    "avatar": {
+      "border-width": { "value": "{ams.border-width.md}" },
```

Comment:
Do we want to use a border? If you make this work with an image you'll lose the with and height.

Reply:
Borders have been removed in latest commit due to the addition of an optional background image.